example/simplegrpcfromconfig/main.go

```diff
@@ -32,42 +31,32 @@ func main() {
 	if err != nil {
 		log.Fatalf("\nerror: %v\n", err)
 	}
-
-	var req = &grpc.Request{
-		Message: &testproto.PredictValuesRequest{
-			PredictionRows: []*testproto.PredictionRow{
-				{
-					RowId: "1",
-				},
-				{
-					RowId: "2",
-				},
+	bytePayload, _ := proto.Marshal(&testproto.PredictValuesRequest{
```

Comment:
From fiber's user perspective (Turing router in our case). Is it possible not to call `Marshall` explicitly?

Reply:
Looks like it's possible, but we can't use generated service code completely and need to introduce some minor modification.

Reply:
If we leave the payload input type as `[]byte`, then it is not possible. If we stay with `interface{}`, it is possible to for Fiber to pass a proto (and marshal it with the default codec) but it defeat the purpose of using a custom codec. 

As long as Fiber does the unmarshal, it needs to know the proto else it will be slower.

Reply:
The dispatcher still need to accept and return `[]byte`. The ideal case is that Turing Router has similar server implementation as `net/http` handler which delegate the marshall and unmarshalling to the server developer. 

The reason I am asking this is because there are 2 performance target with the grpc supports:
1. Can we provides similar performance as the generated grpc code? --> This PR addresses that, we are much closer to this.
2. Can we provides similar or better performance as existing http implementation --> This is the next step.

Achieving number 2 is harder but we'll have to address that since ultimately users will compare against the http performance and latency is usually the main concern when onboarding. The current difference with http implementation (CMIIW) is that additional marshall/unmarshall operation within turing router. If we can minimise , or better eliminate, then we'll be closer to achieving 2.

Reply:
This concern doesn't need to be addressed in this PR as I think it's Turing Router.